lib/flexirest/request.rb

```diff
@@ -224,7 +224,7 @@ def call(explicit_parameters=nil)
 
     def prepare_params
       params = @params || @object._attributes rescue {}
-      if params.is_a?(String) || params.is_a?(Fixnum)
+      if params.is_a?(String) || params.is_a?(Integer)
```

Comment:
As far as I can tell it should be compatible!  [`Integer`][1] exists in 1.9.3.

Here's a loop I've used to test locally:

```bash
#!/bin/bash

while read -r ver; do
  export RBENV_VERSION="$(basename "$ver")"
  ruby -e 'puts %{#{RUBY_VERSION} -- #{[1234,-1234,12345678901234567890,-12345678901234567890,1.1,0].map{|v| [v.class,v.is_a?(Integer)]}}}'
done < <(find ~/.rbenv/versions -mindepth 1 -maxdepth 1 -type d)
```

Which gives me:

```
1.9.3 -- [[Fixnum, true], [Fixnum, true], [Bignum, true], [Bignum, true], [Float, false], [Fixnum, true]]
2.0.0 -- [[Fixnum, true], [Fixnum, true], [Bignum, true], [Bignum, true], [Float, false], [Fixnum, true]]
2.1.0 -- [[Fixnum, true], [Fixnum, true], [Bignum, true], [Bignum, true], [Float, false], [Fixnum, true]]
2.2.0 -- [[Fixnum, true], [Fixnum, true], [Bignum, true], [Bignum, true], [Float, false], [Fixnum, true]]
2.3.1 -- [[Fixnum, true], [Fixnum, true], [Bignum, true], [Bignum, true], [Float, false], [Fixnum, true]]
2.4.1 -- [[Integer, true], [Integer, true], [Integer, true], [Integer, true], [Float, false], [Integer, true]]
```

[1]: http://ruby-doc.org/core-1.9.3/Integer.html